native_client/java/DeepSpeech-jni/org_mozilla_deepspeech_DeepSpeech.cpp

```diff
@@ -0,0 +1,184 @@
+#include "org_mozilla_deepspeech_DeepSpeech.h"
+
```

Comment:
So basically we throw away all the Java SWIG. It means we have to maintain JNI bindings all by ourselves, that's a lot of work.

Reply:
Yes, swig has a lot of overhead.  The way it handles pointers etc is very unperformant.
For bindings that have to be very performant, (such as eg. OpenGL, OpenCV, and also DeepSpeech), it is important to maximize performance and reduce overhead.

Reply:
Have you made any measures yet ?